mapper/docStoreClient.go

```diff
@@ -61,16 +62,38 @@ func (c *httpDocStoreClient) GetContent(uuid, tid string) (*model.DocStoreUppCon
 	return &content, nil
 }
 
+func (c *httpDocStoreClient) ContentExists(uuid, tid string) (bool, error) {
+	docStoreUrl, err := url.Parse(c.docStoreAddress + "/content/" + uuid)
+	if err != nil {
+		return false, fmt.Errorf("failed to parse rawurl into URL structure for docStoreAddress=%v uuid=%v: %v", c.docStoreAddress, uuid, err.Error())
+	}
+	req, err := http.NewRequest(http.MethodGet, docStoreUrl.String(), nil)
+	if err != nil {
+		return false, fmt.Errorf("failed to create request to fetch content for uuid=%v: %v", uuid, err.Error())
+	}
+	req.Host = documentStoreApiHost
```

Comment:
I don't think this is necessary anymore. We no longer do Host based routing since we mo've to Kubernetes. See TODO from the `ContentQuery` function.